### ISSUE TITLE
The ignoreFrom option is not working for tap and doubletap events.

### ISSUE THREAD

**Issue by olivierbelzile** [taye/interact.js#502]
Using version 1.2.8, it seems that tap and doubletap event are always triggered without checking the ignoreFrom option value.

**Comment by wombat86** [comment]
Did you try specifying ignoreFrom using .ignoreFrom('selector') method? For me it didn't work when I provided ignoreForm as the "options" object property. But changing it to setter method instead fixed my problem, although I believe that setter method and "options" should be kept consistent.

**Comment by olivierbelzile** [comment]
Yes, I used the method and it works for most events, but not with the doubletap in my case.

**Comment by taye** [comment]
pointerEvent options are only available from `v1.3.0-alpha.0`.  The options are set like this:

```javascript
interact(target).pointerEvents({
    holdDuration: 600,
    ignoreFrom  : null,
    allowFrom   : null,
    origin      : { x: 0, y: 0 },
});
